src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,75 @@
+package core.basesyntax.service;
+
+import core.basesyntax.exception.UserValidationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static User user;
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
+
+    @BeforeAll
+    static void createValidUser() {
+        user = new User();
+        user.setLogin("validLogin");
+        user.setPassword("validPassword");
+        user.setAge(20);
+    }
+
+    @Test
+    void register_nullUser_notOk() {
```

Comment:
This test case is correct, but make sure you add test cases for all possible edge cases for user parameters as described in the checklist.